components/omega/doc/devGuide/BuildDocs.md

```diff
@@ -0,0 +1,21 @@
+(omega-dev-building-docs)=
+
+# Building the Documentation
+
+As long as you have followed the procedure in {ref}`omega-dev-conda-env` for 
```

Comment:
@grnydawn, I've added some more documentation on the documentation itself to #14 (included here as well). Here is an example of how you can internally link to another section within the documentation by referencing its anchor.  Let me know if this is the type of link you were thinking of in your suggestion.